modules/030-cloud-provider-azure/openapi/config-values.yaml

```diff
@@ -64,3 +59,9 @@ properties:
               type: number
               description: |
                 Disk throughput in `MBps` (limit of a single disk is 256 KiB/s for each provisioned IOPS).
+            tags:
+              type: string
```

Comment:
In configuration we should use object with additionalFields and convert to storage class format in the hook

Reply:
Done, check again pls